quickstart-sda.md

```diff
@@ -0,0 +1,240 @@
+# Quickstart guide to the SDA
+
+This document contains information on how to work with the Sensitive Data Archive (SDA) from the GDI user perspective.
+
+## Brief description of the storage and interfaces stack
+
+The storage and interfaces software stack for the GDI-starter-kit consists of the following services:
+
+| Component     | Description |
+|---------------|------|
+| broker        | RabbitMQ based message broker, [SDA-MQ](https://github.com/neicnordic/sda-mq). |
+| database      | PostgreSQL database, [SDA-DB](https://github.com/neicnordic/sda-db). |
+| storage       | S3 object store, demo uses Minio S3. |
+| mock-oidc     | A python implementation of a mock-oidc in place of LS-AAI. |
+| s3inbox       | Proxy inbox to the S3 backend store, [SDA-S3Proxy](https://github.com/neicnordic/sda-s3proxy). |
+| download      | Data out solution for downloading files from the SDA, [SDA-download](https://github.com/neicnordic/sda-download). |
+| SDA-pipeline     | The ingestion pipeline of the SDA, [SDA-pipeline](https://github.com/neicnordic/sda-pipeline). This comprises of the following core components: `ingest`, `verify`, `finalize` and `mapper`.|
```

Comment:
Should `auth` be added, now what we added it? :)

Reply:
Definitely! I am now thinking to merge this first and add the `auth` documentation to the `sda-quickstart-guide` from the auth compose PR: #21. This way the sequence will be more logical :)